lib/_http_outgoing.js

```diff
@@ -78,11 +78,12 @@ exports.OutgoingMessage = OutgoingMessage;
 OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
-  if (!this.socket) {
+
+  if (!this.socket)
```

Comment:
Agreed. Most major style guides advocate for always having curlies to accompany conditional logic, length irrelevant.